doc/swagger.json

```diff
@@ -614,6 +628,36 @@
                 }
               }
             }
+          },
+          "400": {
+            "description": "User can not be unassigned from the test center with given role",
```

Comment:
Well, it's the same thing rephrased, "user can not be unassigned, because he is not assigned". Would you suggest to make the response description the same as response `errorMsg`?